### ISSUE TITLE
more than 511 leds

### ISSUE THREAD

**Issue by shaystings** [psieg/Lightpack#561]
I know I've seen it here b4 but I can't find it sorry.

Back story I currently have a tensy and a octows2811 to drive 506 LEDs. I have good performance and was contemplating going from 96led/m to 144led/m when I move my tv.  I believe that this should put me close to 760 LEDs. This shouldn't be a problem to drive because I would be running ~100leds per chanel. Any assistance would be much appreciated.  I just want to know it works b4 I buy more LEDs and start the project again.

**Comment by zomfg** [comment]
does not baud well (pun intended) unless you build yourself #382 
https://github.com/psieg/Lightpack/blob/8fad33e208b3044d47cb131534b616c5c3a818df/Software/src/enums.hpp#L102-L119

and bonus math https://www.partsnotincluded.com/calculating-adalight-framerate-limits/

**Comment by shaystings** [comment]
Yes 382 is the post I was remembering , what did you search to find it? Nice dad pun. At present my LEDs are updating at at least 120 fps. The counter isn't accurate If i render 120 frames and use my phone to record the slo mo it keeps up with onscreen images. If 120 is my max frame rate and I ad 33% more LEDs I should be at worse updating at about 90fps which should be fine. Again I'm not using a standard device I am using an octows2811 this uses 8 parallel light strips. The link is a short video of the the frame skip test the white block moves each frame at 120hz. the video is in slo mo. 

https://user-images.githubusercontent.com/76628722/205314963-3d856b96-a42f-4395-9cd3-dae4430dffff.mp4

the test can be found here https://www.testufo.com/frameskipping

As for issue 382 how do I compile the code?  Not sure Gravator ever got instructions for that.

**Comment by zomfg** [comment]
> Yes 382 is the post I was remembering , what did you search to find it?

https://github.com/psieg/Lightpack/issues?q=is%3Aissue+is%3Aopen+511

> Again I'm not using a standard device I am using an octows2811 this uses 8 parallel light strips. 

That probably helps with keeping Tled down (as per that blog link I posted)

> If 120 is my max frame rate and I ad 33% more LEDs I should be at worse updating at about 90fps which should be fine. 

looks like at least some teensy boards can keep up with pretty high baud rates https://www.pjrc.com/teensy/td_uart.html

> As for issue 382 how do I compile the code? Not sure Gravator ever got instructions for that.

https://github.com/psieg/Lightpack#prismatik-build-instructions-for-windows YMMV

**Comment by shaystings** [comment]
> > Yes 382 is the post I was remembering , what did you search to find it?
> 
> https://github.com/psieg/Lightpack/issues?q=is%3Aissue+is%3Aopen+511

I really should learn to search better I did look for most of the words. good job. 

> 
> > Again I'm not using a standard device I am using an octows2811 this uses 8 parallel light strips.
> 
> That probably helps with keeping Tled down (as per that blog link I posted)
> 
> > If 120 is my max frame rate and I ad 33% more LEDs I should be at worse updating at about 90fps which should be fine.
> 
> looks like at least some teensy boards can keep up with pretty high baud rates https://www.pjrc.com/teensy/td_uart.html
> 

I have a 4.0 it should work very fast. I had a knock off Arduino b4 and it only ran at like 6 fps with  ~311 leds and looked terrible. The upgrade was very good letter I bought a new TV and upgraded to 96led/m

> > As for issue 382 how do I compile the code? Not sure Gravator ever got instructions for that.
> 
> https://github.com/psieg/Lightpack#prismatik-build-instructions-for-windows YMMV

Thanks I'll give it a try I haven't read over the instructions for a few years sorry for the dumb question. You have been very helpful. thanks
